signbank/dictionary/update.py

```diff
@@ -82,6 +82,10 @@ def update_gloss(request, glossid):
             gloss.assigned_user_id = value if value and value.strip() != '' else None
             gloss.save()
             newvalue = gloss.assigned_user.get_full_name() if gloss.assigned_user else "None"
+        elif field == 'signer':
```

Comment:
Let's look at this together tomorrow. I don't mind this, but I think this functionality (handling empty strings and setting it to `None` is something we might want to support for any foreign key field choice, in which case we should see if we can figure out why `update.py` line 184 didn't work. Does it just need to be changed to `FieldChoice.objects.filter(value)[0] else None`?